iwyu_path_util.cc

```diff
@@ -149,6 +149,9 @@ string NormalizeFilePath(const string& path) {
   string result = CanonicalizeFilePath(path);
   while (StripLeft(&result, "./")) {
   }
+  // Don't use StripRight as we might have 'foo/../' here
+  while (EndsWith(result, "/./"))
```

Comment:
Nope. We only have forward slashes at this point (`CanonicalizeFilePath` does this)